clang/lib/CIR/Lowering/DirectToLLVM/LowerToLLVM.cpp

```diff
@@ -810,6 +810,17 @@ class CIRConstantLowering
         return mlir::failure();
       }
       attr = denseAttr.value();
+    } else if (const auto structAttr =
+                   op.getValue().dyn_cast<mlir::cir::ConstStructAttr>()) {
+      // TODO(cir): this diverges from traditional lowering. Normally the
+      // initializer would be a global constant that is memcopied. Here we just
+      // define a local constant with llvm.undef that will be stored into the
+      // stack.
```

Comment:
Can you create an issue for this? It's important we do our due-diligence for LLVM differing from CIR, so we can clearly later evaluate!